### ISSUE TITLE
k8s upgrade from 1.10 to 1.13: kube-proxy timed out waiting for the condition

### ISSUE THREAD

**Issue by markito** [kubernetes/minikube#3813]
```
minikube start --cpus 4 --memory 8192
😄  minikube v0.34.1 on darwin (amd64)
👍  minikube will upgrade the local cluster from Kubernetes 1.10.0 to 1.13.3
💡  Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
🔄  Restarting existing virtualbox VM for "minikube" ...
⌛  Waiting for SSH access ...
📶  "minikube" IP address is 192.168.99.108
🐳  Configuring Docker as the container runtime ...
✨  Preparing Kubernetes environment ...
💾  Downloading kubeadm v1.13.3
💾  Downloading kubelet v1.13.3
🚜  Pulling images required by Kubernetes v1.13.3 ...

🔄  Relaunching Kubernetes v1.13.3 using kubeadm ...
⌛  Waiting for kube-proxy to come back up ...

💣  Error restarting cluster: restarting kube-proxy: waiting for kube-proxy to be up for configmap update: timed out waiting for the condition

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new``` 
```

minikube logs: https://gist.github.com/markito/438772bea68294de4b7bede0d8deec8d/revisions
minikube version: v0.34.1
Virtual box: Version 6.0.0 r127566 
OS: MacOS

**Comment by tstromberg** [comment]
Thanks for the bug report! Sorry to hear about your experience here. Do you mind checking if minikube v0.35.0 is able to start this environment? We've changed the code that blocks on kube-proxy to wait longer and give a bit more detail. 

If v0.35.0 does fail, do you mind including `minikube logs` from it. Thanks for helping us nail down this issue!

**Comment by tstromberg** [comment]
Thanks for the bug report! I can confirm that we fixed this in v0.35.0:

```
👍  minikube will upgrade the local cluster from Kubernetes 1.10.0 to 1.13.4
...
⌛  Waiting for pods: apiserver proxy etcd scheduler controller addon-manager dns
📯  Updating kube-proxy configuration ...
...
🏄  Done! Thank you for using minikube!
```

Please let me know if you see it again.
